packages/app/src/specs/SpecsList.vue

```diff
@@ -67,7 +67,7 @@
           :id="getIdIfDirectory(row)"
           :key="row.index"
           :data-cy="row.data.isLeaf ? 'spec-list-file' : 'spec-list-directory'"
-          :data-cy-row="row.data.data?.relative"
+          :data-cy-row="row.data.data?.baseName"
```

Comment:
Just wanted to note that using baseName here doesn't guarantee uniqueness like relative did. Doesn't seem like it impacts any current tests, though.

Reply:
Yep - I think it's probably fine for now, actually updating the paths is a bit of scope creep, and changing the paths dynamically for the assertion seems like more technical debt. This seems like a reasonable trade-off.